code/index.html

```diff
@@ -3,28 +3,143 @@
+        <source scr="https://images.unsplash.com/photo-1526723476341-307183286978?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1650&q=80" media="(max-width: 900)">
+        <img class="header-image" src="https://images.unsplash.com/photo-1526723476341-307183286978?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1650&q=80" alt="Hero Image">
+      </picture>
+      <h1 class="header-text">Travel the World</h1>
+      <h3 class="header-under-text">Find Inspiration for Your Next Post-Covid Trip</h3>
+      <a class="hero-btn" href="https://www.momondo.se/?ispredir=true">Book Your Trip Now!</a>
+      <nav class="navbar">
+        <div>
+          <img class="logo" src="https://picsvg.com/example/owl.svg" alt="Logo">
+        </div>
+        <div class="hamburger" id="hamburger">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+
+        <div class="menu">
+          <a href="https://images.unsplash.com/photo-1584005609794-ed5c60133802?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2498&q=80">ADVENTURE</a>
+          <a href="https://images.unsplash.com/photo-1512552288940-3a300922a275?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2102&q=80">RELAX</a>
+          <a href="https://images.unsplash.com/photo-1504896287989-ff1fbde00199?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=2390&q=80">GETAWAY</a>
+        </div>
+      </nav>
+    </header>
+
     <!-- Signup form -->
+    <div class="form-faq-container">
+      <section>
+        <form class="form" action="https://httpbin.org/anything" method="post">
+            <h3>Create an Account</h3>
+            <input class="input input-box" type="text" name="name" placeholder="Name" required>
+            <input class="input input-box" type="email" name="email" placeholder="Email" required>
+            <input class="input input-box" type="password" name="password" placeholder="Password" required>
+            <input class="input input-box" type="password" name="comfirmed password" placeholder="Confirm Password" required>
+            <label for="form-select"> What travel floats your boat?</label>
+            <select id="form-select" name="form-select" class="form-select">
+              <option value="adventure">Adventure</option>
+              <option value="romantic">Romantic</option>
+              <option value="luxury">Luxury</option>
+              <option value="backpacking">Backpacking</option>
+              <option value="family">Family</option>
+            </select>
+            <p>Do you want to recieve our Newsletter?</p>
+            <div class="radio">
+              <label for="radio-1">Yes!</label>
+              <input name="radio" type="radio">
```

Comment:
Ups, I think you forgot to connect input with labels. If you aim for `id-for` pair, then you should add proper `id` attribute to inputs :

```
<label for="radio-1">Yes!</label>
<input id="radio-1" name="radio" type="radio">
```